filesystems/s3/src/main/scala/cromwell/filesystems/s3/batch/S3BatchIoCommand.scala

```diff
@@ -99,6 +101,7 @@ sealed trait S3BatchHeadCommand[T] extends S3BatchIoCommand[T, HeadObjectRespons
   */
 case class S3BatchSizeCommand(override val file: S3Path) extends IoSizeCommand(file) with S3BatchHeadCommand[Long] {
   override def mapResponse(response: HeadObjectResponse): Long = response.contentLength
+  override def commandDescription: String = s"S3BatchSizeCommand file '$file'"
```

Comment:
TOL one could probably move this `def` up into the trait and avoid having to implement it in every sub-class. OTOH I wouldn't suggest going back and retrospectively doing this:
 
```
override def commandDescription: String = s"${getClass.getSimpleName} file '$file'"
```

Reply:
That is a valid point. Do you have any strong objections to the current approach being merged as it is?

Reply:
Yeah - merging as-is is fine with me